apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -111,33 +117,53 @@ export const NewQuizScreen = () => {
             <View style={localStyles.container}>
               <BigButton
                 buttonColor={colors.lightGrey}
-                textColor={colors.black}
+                textColor={
+                  selectedQuestionType === 'Multiple Choice'
+                    ? colors.white
```

Comment:
Just for future considering, white is not clear

Reply:
You are right, the reason I use white is that Muke hasn't provided the colour palette for the button selection yet